fastlane/Appfile

```diff
@@ -1,7 +1,8 @@
-app_identifier "" # The bundle identifier of your app. (ex: "com.Wolox.Wolox")
-apple_id "ios-si@wolox.com.ar" # Your Apple email address. (ex: "yourName@wolox.com.ar")
+# Your Apple email address. (ex: "yourName@wolox.com.ar")
+apple_id "ios-si@wolox.com.ar"
 
-team_id ""  # Developer Portal Team ID. (ex: "RDFGA2FGMD")
+# Developer Portal Team ID. (ex: "RDFGA2FGMD")
+team_id "7XS5GQGS29"
 
 # you can even provide different app identifiers, Apple IDs and team names per lane:
```

Comment:
Maybe this should be erased too? Since app_identifier no longer exists.

Reply:
👍